src/stores/EndpointConfig/Store.ts

```diff
@@ -87,11 +86,9 @@ const getInitialState = (
                 state.customErrors = val;
                 state.customErrorsExist = hasLength(val);
 
-                const { endpointConfig } = get();
```

Comment:
I am hoping that these `get`s are what is causing potential issues. I almost cleaned these up last time I was in here but decided to leave them because they "weren't hurting anything".